src/pages/Timesheet/index.tsx

```diff
@@ -1,18 +1,22 @@
-import React from 'react';
 import Calendar from '@components/Timesheet';
+import Layout from '@components/Layout';
 import styles from './Timesheet.module.css';
+import { useTranslation } from 'react-i18next';
 
 const month = new Date().toLocaleString('default', { month: 'long' });
 const year = new Date().getFullYear();
 
 const TimesheetPg = () => {
+    const { t } = useTranslation();
     return (
-        <div className={styles.wrapper}>
-            <h2 className={styles.header}>
-                {month} {year}
-            </h2>
-            <Calendar />
-        </div>
+        <Layout>
```

Comment:
Can you try to move Layout component to wrap the routes, so that every route except login route is wrapped with a layout?

Reply:
I made two Routes, one for Login, and one for all other pages (this one is wrapped with Layout)